cost-analyzer/values-thanos.yaml

```diff
@@ -51,11 +51,15 @@ prometheus:
 thanos:
   store:
     enabled: true
+    grpcSeriesMaxConcurrency: 1
+    blockSyncConcurrency: 1
     resources: 
       requests:
-        memory: "2Gi"
+        memory: "2.5Gi"
```

Comment:
This was a changed that @AjayTripathy and I had talked about before. Since `chunk-pool-size` is 2GB and `index-cache-size` is 250MB, we want our requests to fully cover that total + a little breathing room.